src/custom/state/claim/reducer.ts

```diff
@@ -36,6 +36,7 @@ export const initialState: ClaimState = {
     // claiming
     claimStatus: ClaimStatus.DEFAULT,
     claimedAmount: '',
+    estimatedGas: '',
```

Comment:
Wasn't it there before?
I'm sure I added it.

Well, could have been removed during a merge 🤷

Reply:
🤷🏼‍♂️ 🤷🏼‍♂️ 🤷🏼‍♂️